src/openrct2/actions/GameAction.cpp

```diff
@@ -52,8 +52,21 @@ namespace GameActions
         return factory;
     }
 
+    void Initialize()
+    {
+        static bool initialized = false;
+        if (initialized)
+            return;
+
+        Register();
+
+        initialized = true;
+    }
+
     IGameAction * Create(uint32 id)
     {
+        Initialize();
```

Comment:
@IntelOrca already mentioned in the chat this can be probably called from the Context